cypress/integration/distroSettings/project_section.ts

```diff
@@ -0,0 +1,43 @@
+import { clickSave } from "./utils";
+
+describe("project section", () => {
+  beforeEach(() => {
+    cy.visit("/distro/localhost/settings/project");
+  });
+
+  it("can update fields and those changes will persist", () => {
+    cy.dataCy("save-settings-button").should(
+      "have.attr",
+      "aria-disabled",
+      "true"
+    );
+
+    // Update fields.
+    cy.dataCy("clone-method-select").contains("Legacy SSH");
+    cy.dataCy("clone-method-select").click();
+    cy.contains("OAuth").click();
```

Comment:
You might be able to use [`cy.selectLGOption`](https://github.com/evergreen-ci/spruce/blob/bd3fe088a3a6e926e0d398879ce05adecbf87986/cypress/support/commands.ts#L101-L105) here